### DISCUSSION TITLE
Forms of Collaboration > Arrangement > Coupled Collaboration

### DISCUSSION THREAD

**Discussion by giscus** [pgpais/live-framework-for-cooperative-games#90] [Framework Discussions]
# Forms of Collaboration > Arrangement > Coupled Collaboration

https://www.lfcooperativegames.com/reports/new

**Comment by hg65we** [comment]
Each ship type is suited for different situations/tasks, meaning that each player/set of players playing with the same type of ship will/should try to do the task better suited for him/them with the same goal of winning the match
